srcpkgs/pipewire/template

```diff
@@ -81,6 +84,7 @@ libspa-bluetooth_package() {
 	short_desc+=" - bluetooth plugins"
 	pkg_install() {
 		vmove usr/lib/spa-0.2/bluez5
+		vmove etc/pipewire/media-session.d/with-bluetooth
```

Comment:
Nit: `touch $PKGDESTDIR/etc/...` here and avoid having to deal with the file in two different places?